### PR TITLE
Release/0.9.0 - all changes

### PR DESCRIPTION
**Pull request by shakeelmohamed**
Please review @glennblock!

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,29 @@
 # Splunk HTTP Event Collector Stream for Bunyan (beta)
```
Comment (glennblock) [inline CHANGELOG.md]: We should drop using the "beta" term here. .8 was the beta milestone.

File: examples/manual_batching.js [hunk 2]
```diff
@@ -32,16 +29,12 @@ var bunyan = require("bunyan");
...
  */
 var config = {
-    token: "your-token-here",
-    host: "localhost",
-    path: "/services/collector/event/1.0",
-    protocol: "https",
-    port: 8088,
-    level: "info",
-    autoFlush: false
+    token: "56A70C3F-80C1-4879-B11F-BF043B52753F",//"your-token-here",
```
Comment (glennblock) [inline examples/manual_batching.js]: shouldn't value here just be "your-token-here"?

**Comment by glennblock** [comment]
Overall LGTM, though a few comments

File: examples/manual_batching.js (hunk 2 above)
Comment (shakeelmohamed) [inline examples/manual_batching.js]: good catch!

**Pull request merged**